packages/storybook-html/src/components/hero/hero.docs.mdx

```diff
@@ -33,15 +33,41 @@ import StatusBadge from '../../../.storybook/custom/components/StatusBadge';
 <div class="uq-hero uq-hero--feature-image">  
   <div class="uq-hero__image">
     <div class="uq-hero__image--small">
-      <img src="/images/hero/hero-example--small.jpg" />
+      <img src="/images/hero/hero-example--small.jpg" alt="A postgraduate student using a laptop" />
```

Comment:
alt text > is this decorative or part of the information for the title?

Reply:
The current solution works, but consider the decriptions on this page 
https://www.w3.org/WAI/tutorials/images/decorative/

and this 

Decorative images can be hidden from screen readers in multiple ways:

using a null alt attribute
using ARIA role="presentation"
using a CSS background image

A future enhancement might also be to make the image a background image as described.